src/style-spec/util/properties.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+export function isPropertyFunction(spec: Object): boolean {
+    const dataDrivenTypes = new Set(['data-driven', 'cross-faded-data-driven']);
```

Comment:
I don't think we can use Set / Map, since they'd require polyfills for IE 😢

Reply:
Ah, this would have showed up in the style-spec build test, but it's not getting run on CI 😬 https://github.com/mapbox/mapbox-gl-js/issues/6433